### PR TITLE
[luci-interpreter] Introduce Neg kernel

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce Neg kernel to enable CircleNeg IR evaluation.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #5857, from draft #5854

File: compiler/luci-interpreter/src/kernels/Neg.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+void Neg::evalFloat() const
+{
+  tflite::reference_ops::Negate(getTensorShape(input()), getTensorData<float>(input()),
+                                getTensorShape(output()), getTensorData<float>(output()));
+}
+
+void Neg::evalQuantized() const
+{
+  // TODO: implement this
+  throw std::runtime_error("TODO: Support Neg::evalQuantized");
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Neg.cpp:63]: Throwing an exception with "TODO" message seems not natural. How about removing `evalQuantized()` or changing the error message?

File: compiler/luci-interpreter/src/kernels/Neg.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+#include "core/Kernel.h"
+#include <vector>
+
+namespace luci_interpreter
+{
+namespace kernels
+{
+
+class ChannelQuantMultipliers;
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/Neg.h:28]: This seems unnecessary

**Review by jinevening — APPROVED** [review]
LGTM

**Review by struss — APPROVED** [review]
LGTM

**Pull request merged**